fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheAnalyzer.java

```diff
@@ -142,11 +142,12 @@ private void checkCacheConfig() {
                 enableSqlCache = true;
             }
         }
-        if (Config.cache_enable_partition_mode) {
-            if (context.getSessionVariable().isEnablePartitionCache()) {
-                enablePartitionCache = true;
-            }
-        }
+        enablePartitionCache = false;
+        // if (Config.cache_enable_partition_mode) {
+        //     if (context.getSessionVariable().isEnablePartitionCache()) {
+        //         enablePartitionCache = true;
+        //     }
+        // }
```

Comment:
remove useless code, not comment them?

Reply:
> remove useless code, not comment them?

I think the readers can use git blame to find this pr

Reply:
maybe we will revert this comment?